pom.xml

```diff
@@ -160,7 +160,7 @@
     <gatling.version>3.6.1</gatling.version>
     <guava.version>31.0.1-jre</guava.version>
     <hadoop.version>3.3.1</hadoop.version>
-    <iceberg.version>0.12.0</iceberg.version>
+    <iceberg.version>0.12.1</iceberg.version>
```

Comment:
why do we need to upgrade iceberg version when it cannot work with current Nessie ?
Doesn't it make sense to upgrade once [iceberg side PR](https://github.com/apache/iceberg/pull/3257) is merged ?

Reply:
we have a circular dependency in the project with Nessie+Iceberg. The SQL extensions depend on `iceberg.version`, which uses a Nessie version older than the current one, so this is independent from the PR you mentioned

Reply:
ok. got it. 

Now for this `iceberg.version` to work we are keeping `nessie-client` @ 0.9.2 in our pom even though we are in 0.12.1 version.

@rymurr , @snazy , @nastra :  Should we move this `ITNessieStatements` to Nessie module in iceberg code ? so that Nessie code will be clean and always points to latest version for client as-well ?

Reply:
@ajantha-bhat `client.nessie.version` points to 0.9.2 because that's the version that is being used by iceberg 0.12.1 currently. Also we can't move `ITNessieStatements` to Iceberg because the SQL extensions are part of the Nessie codebase, so the test should also be in the Nessie codebase